kin-devplatform-sdk/src/test/java/kin/devplatform/data/blockchain/BlockchainSourceImplTest.java

```diff
@@ -113,12 +103,27 @@ private void resetInstance() throws Exception {
 		blockchainSource = BlockchainSourceImpl.getInstance();
 	}
 
+	@Test
+	public void kinClient_has_no_account_do_not_init_account() {
+		verify(kinClient, never()).hasAccount();
+		verify(kinClient, never()).getAccount(anyInt());
+	}
+
+	@Test
+	public void kinClient_has_account_init_account_with_correct_index() throws Exception {
```

Comment:
what is the scenario that the client has an account but the init is with some wrong index?